aws/logs_monitoring/lambda_function.py

```diff
@@ -655,6 +662,29 @@ def extract_metric(event):
     except Exception:
         return None
 
+def extract_legacy_metric(message):
+    message = message.strip()
+    values = message.split("|")
+    if len(values) != 6:
+        return None
+    _, ts, value, type, name, tags = values
+    
+    timestamp = int(float(ts))  # float to catch decimals
```

Comment:
Our docs says it should always be in seconds https://docs.datadoghq.com/serverless/custom_metrics?tab=python#deprecated-using-cloudwatch-logs?

Reply:
I adapted the logic from the crawler, which had this edge case as well.